### ISSUE TITLE
feature: scroll offset?

### ISSUE THREAD

**Issue by sharethewisdom** [akrennmair/newsbeuter#191]
Vim's setting `scrolloff` specifies the *minimal number of screen lines to keep above and below the cursor*

It would be nice to have such a setting for Newsbeuter in either list or pager views.

**Comment by simonnagl** [comment]
I like this idea, too.

**Comment by sharethewisdom** [comment]
Yes, it's my personal opinion that the added value and potential complexity in the code would meet the design purpose. I'm not a good programmer myself, so I hope someone will respond that has the experience to patch this properly and start running some tests. From what I can find, maybe some inspiration can be obtained from [Cmus's scroll offset implementation](https://github.com/cmus/cmus/blob/0876efa76c366313071fa5bf56089428c0a3cddd/window.c#L96)? I hope it helps.

**Comment by sassmann** [comment]
mutt has a setting called `menu_scroll`. From the docs:
When set, menus will be scrolled up or down one line when you attempt to move
across a screen boundary. If unset, the screen is cleared and the next or
previous page of the menu is displayed (useful for slow links to avoid many
redraws).

The default for mutt is unset, which is a nice setting for going through large amounts of mail. Basically the same applies to newsfeeds. This would be worthwhile addition/alternative to vims `scrolloff`.
Ideally we would have both options to choose from in newsbeuter.
